packages/collections/graphql/schema.js

```diff
@@ -13,5 +13,37 @@ type mutations {
     documentId: ID!
     collectionName: String!
   ): CollectionItem
+
+  clearCollection(
+    collectionName: String!
+  ): Collection!
+
+
+  enableProgressTracking: User!
+  # disabling tracking also deletes all items
+  disableProgressTracking: User!
+  clearProgress: Collection!
+
+
+  upsertDocumentProgress(
+    documentId: ID!
+    percentage: Int!
```

Comment:
Float?
(because it would be more accurate that a rounded Int when we need to fall back to percentage for restoring the position)

Reply:
done